tests/storage/largeblock/test_largeblock_sr_crosspool_migration.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from tests.storage import cold_migration_then_come_back, live_storage_migration_then_come_back
+
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host >= 8.2 with an additional unused 4KiB disk for the SR.
```

Comment:
It will be >= 8.3 at first. If we let this test execute on 8.2 then we can't merge the PR until the driver is available in XCP-ng 8.2.